src/__tests__/element-queries.js

```diff
@@ -35,65 +35,65 @@ test('get throws a useful error message', () => {
   } = render('<div />')
   expect(() => getByLabelText('LucyRicardo'))
     .toThrowErrorMatchingInlineSnapshot(`
-"Unable to find a label with the text of: LucyRicardo
+    "Unable to find a label with the text of: LucyRicardo
 
-<div>
-  <div />
-</div>"
-`)
+    <div>
+      <div />
+    </div>"
+  `)
   expect(() => getByPlaceholderText('LucyRicardo'))
     .toThrowErrorMatchingInlineSnapshot(`
-"Unable to find an element with the placeholder text of: LucyRicardo
+    "Unable to find an element with the placeholder text of: LucyRicardo
 
-<div>
-  <div />
-</div>"
-`)
+    <div>
+      <div />
+    </div>"
+  `)
   expect(() => getByText('LucyRicardo')).toThrowErrorMatchingInlineSnapshot(`
-"Unable to find an element with the text: LucyRicardo. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
+    "Unable to find an element using the provided matcher.The matcher is - LucyRicardo This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.
```

Comment:
This is less descriptive to me. Before I knew it was "text". Now it sounds like there's named matcher called "LucyRicardo" when it's just text.